nautobot_floor_plan/signals.py

```diff
@@ -0,0 +1,34 @@
+"""Signals for the Floor Plan App."""
+
+from django.apps import apps as global_apps
+from django.conf import settings
+
+PLUGIN_SETTINGS = settings.PLUGINS_CONFIG["nautobot_floor_plan"]
+
+MODEL_NAME = "FloorPlanTile"
```

Comment:
This feels like it should be a scoped variable instead of a global variable for all possible future signals - or at least namespaced better.